interface/src/avatar/MyAvatar.cpp

```diff
@@ -4566,6 +4566,13 @@ float MyAvatar::getAvatarScale() {
     return getTargetScale();
 }
 
+// Gets the scale for the avatar that makes the avatar's eye height match the user's real-world eye height
+// (derived from the 'User real world height' setting).
```

Comment:
```suggestion
// (derived from the 'User real world height' Interface setting).
```
It wasn't clear to me whether the setting was in Interface or, say, the Oculus driver.